### PR TITLE
fix types

### PR DESCRIPTION
**Pull request by Rebal67**

### CODE REVIEW

File: src/resolver.ts [hunk 1]
```diff
@@ -165,6 +165,7 @@ export interface VerificationMethod {
   publicKeyMultibase?: string
   blockchainAccountId?: string
   ethereumAddress?: string
+  conditionWeightedThreshold?: any[]
```
Comment (theblockstalk) [inline src/resolver.ts]: maybe good to add a line
```
// TODO add other properties of VerifiableConditions: threshold, conditionThreshold, conditionAnd, conditionOr

Comment (theblockstalk) [inline src/resolver.ts]: change `any` to the type we expect

**Review by theblockstalk — COMMENTED** [review]
package-lock.json and yarn.lock should not change in this PR. please remove them

File: src/resolver.ts (hunk 1 above)
Comment (Rebal67) [reply src/resolver.ts]: added

Comment (Rebal67) [reply src/resolver.ts]: added

Comment (Rebal67) [reply src/resolver.ts]: added type

**Pull request merged**